src/main/java/amazin/model/User.java

```diff
@@ -0,0 +1,76 @@
+package amazin.model;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Entity
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String firstName;
+    private String lastName;
+    @Column(unique = true)
+    private String email;
+    private String password;
+    private String passwordConfirmation;
+    @ManyToMany
+    private Set<Role> roles;
```

Comment:
In which case will Users have more than 1 role?

Reply:
More so for future things but they will have user and admin roles at a maximum for now